addons/payment_stripe/static/src/js/payment_form.js

```diff
@@ -42,7 +43,250 @@ odoo.define('payment_stripe.payment_form', require => {
         },
     };
 
+    const stripeExpressCheckoutMixin = {
+
+        /**
+         * Prepare the express form of Stripe for direct payment.
+         *
+         * @override method from payment.express_form
+         * @private
+         * @param {string} provider - The provider of the selected payment option's acquirer
+         * @param {number} paymentOptionId - The id of the selected payment option
+         * @return {undefined}
+         */
+        _prepareExpressCheckoutForm: function (provider, paymentOptionId) {
+            if (provider !== 'stripe') {
+                return this._super(...arguments);
+            }
+            console.log(paymentOptionId);
+            // Check if instantiation of the drop-in is needed
+            if (this.StripeExpress && this.StripeExpress.acquirerId === paymentOptionId) {
+                return Promise.resolve(); // Don't re-instantiate if already done for this acquirer
+            }
+            var self = this;
+            self._rpc({
+                // Create transaction
+                route: '/payment/stripe/publishable_key',
+                params: {acquirer_id:paymentOptionId},
+            }).then(stripe_publishable_key => {
+                const stripeJS = Stripe(stripe_publishable_key);
+                var isShippingInformationRequired = self._isShippingInformationRequired()
```

Comment:
This is an addition in master using arrow function and const, why is it using callbacks and var?

Reply:
Actually, it is a work in progress and I'm not really at ease with javascript :sweat_smile: Thanks for this remark, I'll look into it :)